Mathlib/CategoryTheory/Sites/CompatibleSheafification.lean

```diff
@@ -58,7 +58,7 @@ Use the lemmas `whisker_right_to_sheafify_sheafify_comp_iso_hom`,
 `to_sheafify_comp_sheafify_comp_iso_inv` and `sheafify_comp_iso_inv_eq_sheafify_lift` to reduce
 the components of this isomorphisms to a state that can be handled using the universal property
 of sheafification. -/
-noncomputable def sheafifyCompIso : J.sheafify P ⋙ F ≅ J.sheafify (P ⋙ F) :=
+noncomputable def sheafifyCompIso : J.plusPlus P ⋙ F ≅ J.plusPlus (P ⋙ F) :=
```

Comment:
Sorry that it took me a long time before reviewing this PR.

I would much prefer if this can still be called `sheafify`. Can't that be accomplished using namespaces?

Reply:
As in: does the new `sheafify` really need to be in the `GrothendieckTopology` namespace?

Reply:
Sorry for taking so long to respond! 

I'll change `GrothendieckTopology.plusPlus` back to `GrothendieckTopology.sheafify` and move the new `sheafify` outside the `GrothendieckTopology` namespace. Is there a smart way to do this in VS Code or do I have to go through everything manually?

Reply:
I made the change, now there are probably some places where the old `GrothendieckTopology.sheafify` API is used  where it could be replaced by the new `sheafify` API, but if either one works we can just leave it and change later if desired.